aws/v9.3.9/README.md

```diff
@@ -0,0 +1,24 @@
+# :zap: Giant Swarm Release v9.3.9 for AWS :zap:
+
+**If you are upgrading from 9.3.8, upgrading to this release will not roll your nodes.**
+
+This patch release fixes the problem causing the accidental deletion and reinstallation of Preinstalled Apps (such as CoreDNS) in 9.x.x tenant clusters.
+
+Please upgrade all older clusters to this version in order to prevent downtime. 
```

Comment:
```suggestion
Please upgrade all older clusters to this version in order to prevent possible downtime. 
```

Maybe we should say possible downtime here? But we do need these clusters to be upgraded.